spec/features/form_sections/teacher_training_data/placements/editing_trainee_placements_spec.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Editing a placement", feature_trainee_placement: true do
+  background do
+    given_i_am_authenticated
+    given_a_trainee_exists
+  end
+
+  scenario "editing an existing placement" do
+    given_i_have_one_placement
+    and_a_school_exists
+
+    and_i_am_on_the_confirm_placement_details_page
+    when_i_click_on_the_change_link
+    then_i_am_taken_to_the_placement_edit_page
+    and_it_is_prepopulated_with_existing_placement
+    when_i_select_an_existing_school
+
+    and_i_click_continue
+    then_i_see_the_confirmation_page
+    and_i_see_the_updated_placement
+  end
+
+private
+
+  def and_it_is_prepopulated_with_existing_placement
+    expect(page.find_by_id("schools-autocomplete-element")["data-default-value"]).to have_text(trainee.placements.first.school.name)
```

Comment:
is there a better way to find this element? nice to avoid `page.find_by_id` if we can.

Reply:
No, it tied to the JavaScript implementation.